plugins/sql-language/src/main/java/org/polypheny/db/sql/SqlProcessor.java

```diff
@@ -169,7 +166,7 @@ public Pair<Node, AlgDataType> validate( Transaction transaction, Node parsed, b
         } catch ( Exception e ) {
             log.error( "Exception while validating query", e );
             String message = e.getLocalizedMessage();
-            throw new GenericRuntimeException( message == null ? "null" : message, -1, "", AvaticaSeverity.ERROR );
+            throw new GenericRuntimeException( message == null ? "null" : message, -1, "", "Sever error encountered" );
```

Comment:
Can we improve this message (see log.error above)

Reply:
Adjusted, to show meaningful error if possible or message, which points to error in doing so.